common/src/main/java/com/microsoft/hydralab/common/entity/common/TestRun.java

```diff
@@ -18,9 +17,9 @@
 
 @Data
 @Entity
-@Table(indexes = {
-        @Index(name = "task_id_index", columnList = "test_task_id", unique = false)})
-public class DeviceTestTask implements Serializable {
+@Table(name = "device_test_task", indexes = {
```

Comment:
@zhou9584 please help check whether the table name is correct, thanks!

Reply:
The table name is correct. We can also transfer the data from old table to the new table when deploying new package.

Reply:
Thanks, please help resolve this conversation if you think it's fine.

Reply:
@zhou9584